### ISSUE TITLE
Selecting "comments" errors with b.sprintf.parse is not a function

### ISSUE THREAD

**Issue by matty** [getsentry/sentry#5515]
Selecting "comments" on an issue causes an error "b.sprintf.parse is not a function" see below for details.

Sentry v8.16.1
Using Docker: Yes

Stacktrace:

```
TypeError: b.sprintf.parse is not a function
  at parse(./app/locale.jsx:45:10)
  at formatForReact(./app/locale.jsx:201:11)
  at format(./app/locale.jsx:210:9)
  at t.formatActivity(./app/views/groupActivity/index.jsx:92:17)
  at formatActivity(./app/views/groupActivity/index.jsx:178:20)
  at map(./app/views/groupActivity/index.jsx:152:34)
  at apply(./app/utils/errorHandler.jsx:8:28)
  at render(~/react/lib/ReactCompositeComponent.js:796:0)
  at _renderValidatedComponentWithoutOwnerOrContext(~/react/lib/ReactCompositeComponent.js:819:0)
  at _renderValidatedComponent(~/react/lib/ReactCompositeComponent.js:361:0)
  at performInitialMount(~/react/lib/ReactCompositeComponent.js:257:0)
  at mountComponent(~/react/lib/ReactReconciler.js:47:0)
  at mountComponent(~/react/lib/ReactChildReconciler.js:122:0)
  at updateChildren(~/react/lib/ReactMultiChild.js:215:0)
  at _reconcilerUpdateChildren(~/react/lib/ReactMultiChild.js:314:0)
  at _updateChildren(~/react/lib/ReactMultiChild.js:301:0)
  at updateChildren(~/react/lib/ReactDOMComponent.js:942:0)
  at _updateDOMChildren(~/react/lib/ReactDOMComponent.js:760:0)
  at updateComponent(~/react/lib/ReactDOMComponent.js:718:0)
  at receiveComponent(~/react/lib/ReactReconciler.js:126:0)
  at receiveComponent(~/react/lib/ReactCompositeComponent.js:751:0)
  at _updateRenderedComponent(~/react/lib/ReactCompositeComponent.js:721:0)
  at _performComponentUpdate(~/react/lib/ReactCompositeComponent.js:642:0)
  at updateComponent(~/react/lib/ReactCompositeComponent.js:544:0)
  at receiveComponent(~/react/lib/ReactReconciler.js:126:0)
  at receiveComponent(~/react/lib/ReactCompositeComponent.js:751:0)
  at _updateRenderedComponent(~/react/lib/ReactCompositeComponent.js:721:0)
  at _performComponentUpdate(~/react/lib/ReactCompositeComponent.js:642:0)
  at updateComponent(~/react/lib/ReactCompositeComponent.js:544:0)
  at receiveComponent(~/react/lib/ReactReconciler.js:126:0)
  at receiveComponent(~/react/lib/ReactChildReconciler.js:110:0)
  at updateChildren(~/react/lib/ReactMultiChild.js:215:0)
  at _reconcilerUpdateChildren(~/react/lib/ReactMultiChild.js:314:0)
  at _updateChildren(~/react/lib/ReactMultiChild.js:301:0)
  at updateChildren(~/react/lib/ReactDOMComponent.js:942:0)
  at _updateDOMChildren(~/react/lib/ReactDOMComponent.js:760:0)
  at updateComponent(~/react/lib/ReactDOMComponent.js:718:0)
  at receiveComponent(~/react/lib/ReactReconciler.js:126:0)
  at receiveComponent(~/react/lib/ReactChildReconciler.js:110:0)
  at updateChildren(~/react/lib/ReactMultiChild.js:215:0)
  at _reconcilerUpdateChildren(~/react/lib/ReactMultiChild.js:314:0)
  at _updateChildren(~/react/lib/ReactMultiChild.js:301:0)
  at updateChildren(~/react/lib/ReactDOMComponent.js:942:0)
  at _updateDOMChildren(~/react/lib/ReactDOMComponent.js:760:0)
  at updateComponent(~/react/lib/ReactDOMComponent.js:718:0)
  at receiveComponent(~/react/lib/ReactReconciler.js:126:0)
  at receiveComponent(~/react/lib/ReactChildReconciler.js:110:0)
  at updateChildren(~/react/lib/ReactMultiChild.js:215:0)
  at _reconcilerUpdateChildren(~/react/lib/ReactMultiChild.js:314:0)
```

**Comment by mattrobenolt** [comment]
Is this still a problem with 8.17? We definitely should have fixed this.

**Comment by matty** [comment]
Correct this is fixed in 8.17. Just upgraded and tested will close this issue.
